src/lsp/cobol_cfg/cfg_options.ml

```diff
@@ -0,0 +1,23 @@
+(******************************************************************************)
+(*                                                                            *)
+(*     Copyright (c) 2021-2023 OCamlPro SAS                                   *)
+(*                                                                            *)
+(*     All rights reserved.                                                   *)
+(*     This file is distributed under the terms of the                        *)
+(*     OCAMLPRO-NON-COMMERCIAL license.                                       *)
+(*                                                                            *)
+(******************************************************************************)
+
+type transformation =
+  | Descendents of int
+  | Neighborhood of int
+
+type t = {
+  hide_unreachable: bool;
+  collapse_fallthru: bool;
+  shatter_hubs: int option;
+  transformation: transformation option;
+  hidden_nodes: int list;
+  split_nodes: int list;
```

Comment:
Note sure what those two last lists correspond to. What do the `int`s correspond to?

Reply:
They correspond to node ids

Reply:
Could you add a comment to specify where these IDs come from? Do they come from a previous CFG construction? (there's an integrity issue here, but that may still be ok for now).